src/Schema/Directives/WithoutGlobalScopesDirective.php

```diff
@@ -0,0 +1,53 @@
+<?php declare(strict_types=1);
+
+namespace Nuwave\Lighthouse\Schema\Directives;
+
+use Illuminate\Database\Eloquent\Builder as EloquentBuilder;
+use Illuminate\Database\Eloquent\Relations\Relation;
+use Illuminate\Database\Query\Builder as QueryBuilder;
+use Nuwave\Lighthouse\Exceptions\DefinitionException;
+use Nuwave\Lighthouse\Support\Contracts\ArgBuilderDirective;
+use Nuwave\Lighthouse\Support\Contracts\ArgDirectiveForArray;
+
+final class WithoutGlobalScopesDirective extends BaseDirective implements ArgDirectiveForArray, ArgBuilderDirective
+{
+    public static function definition(): string
+    {
+        return /** @lang GraphQL */ <<<'GRAPHQL'
+directive @withoutGlobalScopes(
```

Comment:
```suggestion
"""
Removes any number of global scopes from the query builder.

The scopes will only be removed if the client-given value of the argument is truthy.
```
directive @withoutGlobalScopes(
```